challenge_gpsr/src/gpsr.py

```diff
@@ -97,14 +97,13 @@ def _on_request_missing_information(self, description, grammar, target):
                                                            grammar=grammar,
                                                            target=target)
 
-
                 if not self.is_text_valid_input(sentence):
                     self._say_to_user("I don't understand what you're saying, please rephrase")
                     continue
 
                 self.timeout_count = 0
                 correct = True
-                if self.test:
+                if not self.test:
```

Comment:
Why has this changed?

Reply:
This was wrong. If not testing, you are in a real challenge. So you're asking for confirmation.